paste-html-as-github-markdown/info.json

```diff
@@ -2,8 +2,8 @@
   "name": "Paste HTML as GitHub Markdown",
   "identifier": "paste-html-as-github-markdown",
   "script": "paste-html-as-github-markdown.qml",
-  "authors": ["@pbek"],
-  "version": "0.0.1",
+  "authors": ["@pbek"],["@Zorziel"],
```

Comment:
Oh, that's no valid JSON any more. I wonder why my test didn't find that. I just now, after I merged the PR, git the error.

Reply:
Oh, the tests did fail for the PR, I overlooked that...